json/src/lib.rs

```diff
@@ -745,6 +745,34 @@ pub struct ImportMultiOptions {
     pub rescan: Option<bool>,
 }
 
+#[derive(Clone, PartialEq, Eq, Copy, Debug)]
+pub enum ImportMultiRescanSince {
+    Now,
+    Timestamp(u64),
+}
+
+impl serde::Serialize for ImportMultiRescanSince {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        match *self {
+            ImportMultiRescanSince::Now => {
+                serializer.serialize_str("now")
+            }
+            ImportMultiRescanSince::Timestamp(timestamp) => {
+                serializer.serialize_u64(timestamp)
+            }
+        }
+    }
+}
+
+impl Default for ImportMultiRescanSince {
```

Comment:
How useful is the `Default` impl? I'm not sure if there actually is a good default value, it heavily depends on your use case. So it seems a bit opinionated. Ok if useful, otherwise I'd like to avoid it.

Reply:
I had to implement `Default` because `ImportMultiRequest`, which holds `ImportMultiRescanSince`, implements `Default` too.

This allow to construct `ImportMultiRequest`s using pattern of `ImportMultiRequest { some: fields_set, ..Default::default() }`, which is quite useful when there are so many fields.

Reply:
Hmm yeah the upstream doesn't have a default, right? Does time 0 make sense at all? It seems like an easy mistake and your bitcoind will be lagging for hours?

Reply:
Yes, bitcoind requires an explicit `timestamp` field and refuses to import otherwise. I can see how it would make sense to do the same here, but having to manually specify all of `ImportMultiRequest`'s fields when constructing it can be quite a PITA.

Is there a way to enable default values for all of `ImportMultiRequest`'s fields except for `timestamp`?

Reply:
> Is there a way to enable default values for all of ImportMultiRequest's fields except for timestamp?

How about we instead make the field an `Option<ImportMultiRescanSince>` and let bitcoind complain if its missing?

Reply:
Yeah that's also a bit annoying. But more correct. Even though it does suggest that it's optional :D 

How bad/good is the `0` default? It seems that having "now" as default is less painful if accidentally forgotten about. Even though you could argue that then a user doesn't get results and might not know why. Yeah 0 is probably what users would expect when not providing it.

Perhaps a documentation warning would be sufficient and keep the `0` default.

Reply:
Between `now` and `0`, I would go with a default of `0`. I don't think its too bad for a developer-facing API if it gets documented properly. It's more tricky if you're dealing with end-users directly, like bitcoind does.

Reply:
I added a doc comment as follow: "Note: unlike in bitcoind, `timestamp` defaults to 0."

Reply:
Good. Sounds ok.